ivy/functional/frontends/paddle/nn/functional/vision.py

```diff
@@ -135,3 +136,118 @@ def affine_grid(theta, out_shape, align_corners=True):
             base_grid[:, :, :, :, 3] = ivy.full((D, H, W), 1)
             grid = ivy.matmul(base_grid.view((N, D * H * W, 4)), theta.swapaxes(1, 2))
             return grid.view((N, D, H, W, 3))
+
+
+@to_ivy_arrays_and_back
+@with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
+def grid_sample(input, grid, mode, padding_mode):
```

Comment:
the fronend function signature must match the native framework function

```suggestion
def grid_sample(
    x,
    grid,
    mode='bilinear',
    padding_mode='zeros',
    align_corners=True,
    name=None,
):
```